mitosheet/mitosheet/code_chunks/snowflake_import_code_chunk.py

```diff
@@ -36,13 +38,26 @@ def get_description_comment(self) -> str:
 
     def get_code(self) -> Tuple[List[str], List[str]]:
 
-        connection_param_transpiled_code = get_param_dict_as_code(self.connection_params_dict)
-
-        snowflake_connection_code = [
-            f'con = snowflake.connector.connect({connection_param_transpiled_code})',
-            '',
-            'cur = con.cursor()',
-        ]
+        imports = ['import snowflake.connector']
+
+        if not self.credentials_are_in_env:
+            connection_param_transpiled_code = get_param_dict_as_code(self.connection_params_dict)
+            snowflake_connection_code = [
+                f'con = snowflake.connector.connect({connection_param_transpiled_code})',
+                '',
+                'cur = con.cursor()',
+            ]
+        else:
+            imports.append('import os')
+            snowflake_connection_code = [
+                'con = snowflake.connector.connect(',
+                f'    user=os.environ[\'SNOWFLAKE_USERNAME\'],',
+                f'    password=os.environ[\'SNOWFLAKE_PASSWORD\'],',
+                f'    account=os.environ[\'SNOWFLAKE_ACCOUNT\'],',
+                ')',
```

Comment:
This should be the full connection string I think, not just the user, password, and account!